da/src/main/java/org/jboss/bacon/da/DAReportsCli.java

```diff
@@ -2,18 +2,18 @@
 
 import lombok.extern.slf4j.Slf4j;
 import org.jboss.bacon.da.rest.endpoint.ReportsApi;
+import org.jboss.da.model.rest.GAV;
 import org.jboss.da.model.rest.NPMPackage;
 import org.jboss.da.reports.model.api.SCMLocator;
-import org.jboss.da.reports.model.request.AlignReportRequest;
-import org.jboss.da.reports.model.request.BuiltReportRequest;
-import org.jboss.da.reports.model.request.SCMReportRequest;
-import org.jboss.da.reports.model.request.VersionsNPMRequest;
+import org.jboss.da.reports.model.request.*;
```

Comment:
The PNC style kinda dicates that you don't use the catch-all '*' import. Could you please revert it back to explicitly having all the import lines one by one pelase?

Reply:
Sure, done